lib/Application/Controller/IndexController.php

```diff
@@ -88,6 +88,7 @@ private function showIndex(): void
             $statusService = new \Poweradmin\Application\Service\PowerdnsStatusService();
             $serverStatus = $statusService->getServerStatus();
             $pdnsServerStatus = [
+                'display' => $serverStatus['display_name'] ?? 'localhost',
```

Comment:
The default value in default settings is PowerDNS, so it should be default also here instead of localhost.
```
                'display' => $serverStatus['display_name'] ?? 'PowerDNS',
```